code/game/objects/items/implants/implantchair.dm

```diff
@@ -190,4 +190,4 @@
 	brainwash(C, objective)
 	message_admins("[ADMIN_LOOKUPFLW(user)] brainwashed [key_name_admin(C)] with objective '[objective]'.")
 	log_game("[key_name(user)] brainwashed [key_name(C)] with objective '[objective]'.")
-	return TRUE
\ No newline at end of file
+	return TRUE

```

Comment:
honk?

Reply:
`editorconfig` forces new lines on files you touch. This was one that I opened and saved out of habit.